package.json

```diff
@@ -33,7 +33,7 @@
     "test:format": "prettier --list-different './**/*.{js,json,md,ts,scss}'",
     "pretest:lint": "tsc -p tests/tslint/tsconfig.json",
     "test:lint:ts": "tslint -p . -c tslint.json \"src/**/*.ts\" -e \"src/schematics/**\"",
-    "test:lint:styles": "stylelint \"src/**/*.scss\" \"src/**/*.css\"",
```

Comment:
Doesn't this now just ignore CSS files?

Reply:
That was a typo from testing, there is one CSS file left in the project the 404 page which I didnt convert to Sass. I could convert it and then there would be no CSS files left or just add back the lint.

Reply:
I went ahead and renamed the last css file to sass. So there are no plain css files in the project. I can add the lint for css files back if you think its necessary.